LK_proposals/LK Proposal for Collaboration.md

```diff
@@ -0,0 +1,116 @@
+## Proposal for Collaboration within our simulation structure
+
+There exists substantial flexibility in the conduction of adversarial collaboration (AC) (Mellers et al., 2001;  Cowan, 2020; Melloni et al., 2021). The below presented approach tries to incorporate suggestions from the literature, while also introducing deviances due to our use case.
+
+**First round: seperate studies:**
+
+In order to have a starting point of different perspectives, as well as to be able to investigate whether AC shows incremental improvements over preregistration, a preliminary simulation study should be pursued by both researchers on their own.
```

Comment:
I like the idea of investigate whether AC shows incremental improvements over preregistration, however it should be noted that this would only generate anecdotal evidence. If the actual goal of the project is to showcase the functional practicability and conductibility of AC this might not add to this in substantial terms. Nevertheless the effort of implementing an additional AC round along the clear cut between preregistration and execution is low as this clear cut should be present in any case as best practice of simulations in general.

Reply:
I agree that we maybe should just broadly discuss the incrementel improvements in a qualitative way and focus on the general feasability of AC.